### ISSUE TITLE
Not available for flutter ios

### ISSUE THREAD

**Issue by tejbhansahu** [wu9007/qrcode_scanner#97]
qr_scan flutter package is not working in the ios app

**Comment by omarhodrob** [comment]
Yes flutter MissingPluginException(No implementation found for method scan on channel qr_scan) 

not work on channel Stable

**Comment by LampeMW** [comment]
What version/channel of flutter are you on? It's working for me on iOS right now and I'm on the newest stable version.

**Comment by Rodrigolmti** [comment]
In the README file you mention todo iOS documentation, any plans to do that? Because i believe that everyone is having this issue.

**Comment by wu9007** [comment]
iOS is implemented on `master` (qrscan 0.4.0).

Camera / album / path / bytes all go through AVFoundation + Vision. Add `NSCameraUsageDescription` and `NSPhotoLibraryUsageDescription` to `ios/Runner/Info.plist`. Minimum iOS 12.

This is still a one-shot `scan()`. If you need a scan window or continuous scan, use [mobile_scanner](https://pub.dev/packages/mobile_scanner).

Install from git until pub.dev ships 0.4.0:

```yaml
dependencies:
  qrscan:
    git:
      url: https://github.com/wu9007/qrcode_scanner.git
      ref: master
```

Closing as completed.

**Comment by wu9007** [comment]
@tejbhansahu “qr_scan flutter package is not working in the ios app” was the stub. 0.4.0 is a real iOS implementation on the same Dart API.

If `scan()` still no-ops, check `NSCameraUsageDescription` and that you are not on pub 0.3.3. Closing as completed.
